Code/GraphMol/catch_graphmol.cpp

```diff
@@ -1108,3 +1117,248 @@ TEST_CASE("github #2895: acepentalene aromaticity perception ",
     CHECK(smi == "C1=CC2=C3C1=CC=C3C=C2");
   }
 }
+
+TEST_CASE("handling of bondStereoCare in updateQueryProperties") {
+  SECTION("fully specified") {
+    auto mol = R"CTAB(basic test
+  Mrv1810 01292006422D          
+
+  0  0  0     0  0            999 V3000
+M  V30 BEGIN CTAB
+M  V30 COUNTS 4 3 0 0 0
+M  V30 BEGIN ATOM
+M  V30 1 C -7.0316 2.0632 0 0 STBOX=1
+M  V30 2 C -5.6979 2.8332 0 0 STBOX=1
+M  V30 3 O -4.3642 2.0632 0 0
+M  V30 4 F -8.3653 2.8332 0 0
+M  V30 END ATOM
+M  V30 BEGIN BOND
+M  V30 1 1 2 3
+M  V30 2 1 1 4
+M  V30 3 2 1 2 STBOX=1
+M  V30 END BOND
+M  V30 END CTAB
+M  END
+)CTAB"_ctab;
+    REQUIRE(mol);
+    REQUIRE(mol->getBondBetweenAtoms(0, 1));
+    CHECK(mol->getBondBetweenAtoms(0, 1)->getStereo() ==
+          Bond::BondStereo::STEREOE);
+    MolOps::AdjustQueryParameters ps;
+    ps.useStereoCareForBonds = true;
+    MolOps::adjustQueryProperties(*mol, &ps);
+    CHECK(mol->getBondBetweenAtoms(0, 1)->getStereo() ==
```

Comment:
Could you also have a test for useStereoCareForBonds = false?  It's hard to tell in this test if anything actually happened.

Reply:
Comment so I can start a  review

Reply:
`useStereoCareForBonds=false` is the default, so basically all of the other tests use that.